client/src/components/Form/Elements/FormText.vue

```diff
@@ -0,0 +1,144 @@
+<template>
+    <div>
+        <b-alert class="mt-2" v-if="errorMessage" :show="dismissCountDown" variant="info" @dismissed="resetAlert">
+            {{ errorMessage }}
+        </b-alert>
+        <b-row align-v="center">
+            <b-col>
+                <!-- type        <str> "text" or "password" -->
+                <!-- area        <bool> presumably for a textarea instead of input -->
+                <!-- readonly    <bool> -->
+                <!-- color       <str> -->
+                <!-- styleObj    <obj> applied to input element -->
+                <!-- placeholder <str> -->
+                <!-- datalist    <arr> -->
+
+                <!-- multiple <bool>
+                     Allow multiple entries to be created?
+                     Seems to use a textarea currently... line separated values?
+                 -->
+
+                <component
+                    :is="componentName"
+                    :id="id"
+                    :readonly="readonly"
+                    :placeholder="placeholder"
+                    :style="style"
+                    :type="type"
+                    v-model="currentValue"
+                    @change="onInputChange" />
+
+                <datalist v-if="datalist && !area && !multiple" :id="`${id}-datalist`">
+                    <option v-for="d in datalist" value="d"></option>
+                </datalist>
+            </b-col>
+        </b-row>
+    </div>
+</template>
+
+<script>
+export default {
+    props: {
+        value: {
+            // String or Array for multiple
+            required: false,
+            default: "",
+        },
+        id: {
+            type: String,
+            required: false,
+        },
+        type: {
+            type: String,
+            required: false,
+            default: "text",
+            validator: (prop) => ["text", "password"].includes(prop.toLowerCase()),
```

Comment:
This validation does not work here, since the input field is used as fallback for other fields such as `drilldown` `select` in the workflow editor (see: `parameters.js`).

Reply:
Ok I have removed this - I don't think it makes so much sense given the fallback use cases.